src/generic-container/generic-container.ts

```diff
@@ -221,6 +224,16 @@ export class GenericContainer implements TestContainer {
     return this;
   }
 
+  public withLabels(labels: Labels): this {
+    this.labels = { ...labels };
+    return this;
+  }
+
+  public addLabels(labels: Labels): this {
```

Comment:
I am not sure about adding `addX` methods 🙂  
I think that `withX` can satisfy all cases, and halves the API we expose to users, makes changes easier, etc.

Reply:
got it 👍🏼
makes sense